docs/serverless-v2/01-02-serverless-v2.md

```diff
@@ -14,19 +14,19 @@ Serverless v2 replies on [Knative Serving](https://knative.dev/docs/serving/) fo
 
 4. Tekton Controller (TC) detects the new TaskRun CR and reads its definition.
 
-5. Based on the TaskRun CR definition, TC triggers a pipeline that creates an image with lambda definition (Dockerfile). It later pushes this image to the local registry.
+5. Based on the TaskRun CR definition, TC triggers a pipeline that uses [Kaniko](https://github.com/GoogleContainerTools/kaniko/blob/master/README.md) to create a Docker image with lambda definition and publish this image in a Docker registry.
```

Comment:
`in an internal Docker registry`* 

this seems to be the case we will be supporting rn, as agreed with @michal-hudy 

unless it is not needed to be specified, would like an second opinion here

Reply:
answer: nope :D just discussed offline, better not make it so detailed